src/performance/postgres/postgresql_performance_test.py

```diff
@@ -10,13 +10,15 @@ def __init__(self, operations_number: int):
             operations_number, CustomerPostgresRepository(), EventPostgresRepository(), OrderPostgresRepository()
         )
 
-    def present_results(self) -> None:
+    def present_results(self, draw_chart: bool) -> None:
+        database_name = 'PostgreSQL'
+
         customers_table_results = self.test_customers_table()
         events_table_results = self.test_events_table()
         orders_table_results = self.test_orders_table()
 
         print('> PERFORMANCE TESTS RESULTS')
-        print('> DATABASE TYPE: PostgreSQL')
+        print(f'> DATABASE TYPE: {database_name}')
```

Comment:
you make a variable db name, but your files are named after db names. any way to make it as one `present_results` then? Variable `database_name` is not coming from outside of function anyways

Reply:
I created this variable only for not repeating the same string
```
print(f'> DATABASE TYPE: PostgreSQL')
self.draw_bar_chart('PostgreSQL', customers_table_results, events_table_results, orders_table_results)
```